core/domain/exp_services.py

```diff
@@ -943,14 +1198,32 @@ def save_exploration_summary(exp_summary):
 
 
 def delete_exploration_summary(exploration_id):
-    """Delete an exploration summary model."""
+    """Delete an exploration summary model.
+
+    Args:
+        exploration_id: str. The id of the exploration summary to be
+            deleted.
+    """
 
     exp_models.ExpSummaryModel.get(exploration_id).delete()
 
 
 def revert_exploration(
         committer_id, exploration_id, current_version, revert_to_version):
-    """Reverts an exploration to the given version number. Commits changes."""
+    """Reverts an exploration to the given version number. Commits changes.
+
+    Args:
+        committer_id: str. ID of the user who made the commit.
+        exploration_id: str. ID of the exploration to be reverted to the
+            curent version.
+        current_version: float. The present version.
+        revert_to_version: float. The version to which the given exploration 
+            is to be reverted.
+
+    Raises:
+        Exception: The given version does not match the currently-stored
+            exploration model
```

Comment:
... does not match the version of the currently-stored exploration model.

(Add full stop at end, too.)